cmd/parser/parse_test.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/kyma-project/kyma-environment-broker/common/hyperscaler/rules"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+const RULES_TEST_CASES = "rules/test-cases.yaml"
+
+type TestCases struct {
+	Case []*TestCase `yaml:"cases"`
+}
+
+type TestCase struct {
+	Name         string   `yaml:"name"`
+	Rules        []string `yaml:"rule"`
+	ExpectedRule string   `yaml:"expected"`
+}
+
+func (c *TestCases) loadCases() *TestCases {
+	yamlFile, err := os.ReadFile(RULES_TEST_CASES)
+	if err != nil {
+		log.Printf("err while reading a file %v ", err)
+	}
+	err = yaml.Unmarshal(yamlFile, c)
+	if err != nil {
+		log.Fatalf("Unmarshal: %v", err)
+	}
+
+	return c
+}
+
+func (c *TestCases) writeCases() *TestCases {
```

Comment:
The same remark.
We do not need to return anything.

Reply:
https://github.com/ralikio/kyma-environment-broker/commit/90ae72fa30be9065e4f50f51df81f82caeef8cca